grid/grid_network.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import json
+import syft as sy
+from grid.websocket_client import WebsocketGridClient
+
+
+class GridNetwork(object):
+    """  The purpose of the Grid Network class is to control the entire communication flow by abstracting operational steps.
+    
+        Attributes:
+            - gateway_url : network address to which you want to connect
+            - connected_grid_nodes : Grid nodes that are connected to the application.
+    """
+
+    def __init__(self, gateway_url):
+        self.gateway_url = gateway_url
+        self.connected_grid_nodes = {}
+
+    def search(self, *query):
+        """ Search a set of tags across the grid network 
+            
+            Parameters:
+                query : A set of dataset tags
+            Return:
+                tensor_set : matrix of tensor pointers
```

Comment:
If this is a matrix of tensor pointers why is it called tensor_set?

Reply:
Done! (7b5588b)